projects/lights_control/test/test_lights_gpio.c

```diff
@@ -0,0 +1,126 @@
+#include <stdint.h>
+
+#include "event_queue.h"
+#include "gpio.h"
+#include "misc.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+#include "lights_events.h"
+#include "lights_gpio.h"
+#include "lights_gpio_config.h"
+
+#define TEST_LIGHTS_GPIO_MAKE_MASK(num) ((1) << (num))
+
+typedef enum {
+  LIGHTS_ACTION_TURN_OFF = 0,  //
+  LIGHTS_ACTION_TURN_ON,       //
+  NUM_LIGHTS_ACTIONS           //
+} LightsAction;
```

Comment:
Why is this defined in the test? Also, note that test structs and enums should still follow the naming convention.